liquibase-standard/src/main/java/liquibase/database/core/PostgresDatabase.java

```diff
@@ -75,6 +75,10 @@ public PostgresDatabase() {
         super.sequenceCurrentValueFunction = "currval('%s')";
         super.unmodifiableDataTypes.addAll(Arrays.asList("bool", "int4", "int8", "float4", "float8", "bigserial", "serial", "oid", "bytea", "date", "timestamptz", "text", "int2[]", "int4[]", "int8[]", "float4[]", "float8[]", "bool[]", "varchar[]", "text[]", "numeric[]"));
         super.unquotedObjectsAreUppercased = false;
+
+        // Fix for issue 1451
```

Comment:
Could you remove this comment? Git log should be enough to track this.

Reply:
Done.